aws-cpp-sdk-core-tests/aws/auth/AWSHttpResourceClientTest.cpp

```diff
@@ -68,6 +68,8 @@ namespace
         ASSERT_EQ("/path/to/res", mockRequest.GetUri().GetPath());
         ASSERT_EQ(Aws::Http::HttpMethod::HTTP_GET, mockRequest.GetMethod());
         ASSERT_EQ("", result);
+        // Retries should be done when request can't be made at all
+        ASSERT_EQ(5u, mockHttpClient->GetAllRequestsMade().size());
```

Comment:
Could you please add an inline comment here explaining how `5` come up?, E.g. `5/* 1 initial request + 4 retries*/`